fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java

```diff
@@ -152,13 +146,10 @@ public Response addVersion(@HeaderParam("Slug") final String slug) throws Reposi
      *
      * @return List of versions for the object as RDF
      */
-    @SuppressWarnings("resource")
     @GET
     @HtmlTemplate(value = "fcr:versions")
-    @Produces({TURTLE_WITH_CHARSET + ";qs=1.0", JSON_LD + ";qs=0.8", N3_WITH_CHARSET, N3_ALT2_WITH_CHARSET,
-            RDF_XML, NTRIPLES, TEXT_PLAIN_WITH_CHARSET,
-            TURTLE_X, TEXT_HTML_WITH_CHARSET, "*/*"})
-    public RdfNamespacedStream getVersionList() {
+    @Produces({ APPLICATION_LINK_FORMAT, "*/*" })
```

Comment:
Since the LDPCv is both a container and a timemap, we will need support the standard container serializations in addition to 'application/link-format'.
See: https://github.com/fcrepo/fcrepo-specification/pull/244#issuecomment-337663664

Reply:
To clarify, that means for requests for N3 or Turtle I need to send the ldp:contains stuff? But for link-format requests I send this stuff.

Reply:
For N3, Turtle, HTML, JSON-LD, etc requests, you send back all of the normal LDPC triples:
- server-managed triples
- client-provided triples
- ldp:contains
- etc

Yes, for `application/link-format`, you do exactly what you are doing in this PR... with the addition of actual mementos, once we are able to create them.